pkg/networkgraph/entity.go

```diff
@@ -27,10 +30,30 @@ var (
 type Entity struct {
 	Type storage.NetworkEntityInfo_Type
 	ID   string
+
+	// TODO this does not belong here
+	ExternalEntityAddress net.IPNetwork
+	Learned               bool
```

Comment:
Up to now, every EXTERNAL_SOURCE entity pre-existed in central and is only referenced by an Entity.
Now that we _learn_ entities, we need a way to convey the learned attributes.

It does not feel quite right to add `ExternalEntityAddress` and `Learned` here, but I could not find a light and elegant way to store this information, yet.

The alternatives I considered are:
- make use of polymorphism and store the attributes in the specialized entity. This is extremely heavy.
- store the EXTERNAL_SOURCE specifics in a dedicated table and lookup the table when creating the storage object. The entries life-cycle is not trivial, and there is no way to know that an Entity is learned.
- punch a hole all the way back to where Sensor sends the netflow update to add an extra list of learned EXTERNAL_SOURCE entities to send. Then merge before sending.